helm/amko/templates/statefulset.yaml

```diff
@@ -57,6 +57,10 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: metadata.name
+          {{ if .Values.multiClusterIngress.enable }}
+          - name: MCI_ENABLED
```

Comment:
Are we using MCI_ENABLED anywhere ?

Reply:
yes, we are using the flag while initializing the MCI informers.